src/boot/antelope.ts

```diff
@@ -97,6 +97,12 @@ export default boot(({ app }) => {
     ant.config.setIndexerHealthThresholdSeconds(10);
     ant.config.setIndexerHealthCheckInterval(5000);
 
+    // We only allow debug mode if we are not in production or in a sensitive environment
+    const weAreNotInProduction = process.env.NODE_ENV !== 'production';
+    const weAreInLocalhost = document.location.hostname === 'localhost';
+    const weAreInNetlify = document.location.hostname.includes('netlify');
+    ant.config.debug.allowDebugMode(weAreNotInProduction || weAreInLocalhost || weAreInNetlify);
```

Comment:
This is a condition to allow debug mode or not. This is a configuration outside the Antlib so we can put any condition.